### PR TITLE
[consensus] Enable distributed vote scoring in mainnet v63

### PR DESCRIPTION
**Pull request by arun-koshy**
## Description 

Bump to protocol version 63 

## Test plan 

How did you test the new or updated feature?

Has been running nightly in private-testnet and also running successfully in testnet https://metrics.sui.io/goto/KDyf-3kHR?orgId=1

---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [X] Protocol: Enable distributed vote scoring in mainnet v63 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [ ] JSON-RPC: 
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:
- [ ] REST API:

### CODE REVIEW

**Comment by akichidis** [comment]
> https://metrics.sui.io/goto/KDyf-3kHR?orgId=1

@arun-koshy are you targeting v1.35 for this? Unless we are in a hurry do you think we could leave the new scores run maybe for a bit longer?

**Comment by arun-koshy** [comment]
This would be for 1.36

File: crates/sui-protocol-config/src/lib.rs [hunk 1]
```diff
@@ -2820,6 +2821,13 @@ impl ProtocolConfig {
                 62 => {
                     cfg.feature_flags.relocate_event_module = true;
                 }
+                63 => {
+                    cfg.feature_flags.relocate_event_module = true;
```
Comment (mwtian) [inline crates/sui-protocol-config/src/lib.rs:2825]: Fix？

**Review by mwtian — APPROVED** [review]

File: crates/sui-protocol-config/src/lib.rs (hunk 1 above)
Comment (mwtian) [reply crates/sui-protocol-config/src/lib.rs:2825]: cc @arun-koshy

Comment (arun-koshy) [reply crates/sui-protocol-config/src/lib.rs:2825]: Missed this, thanks for pointing it out. @halfprice is going to fix it when he moves this feature flag to v64

**Pull request merged**